src/components/aiAgent/ChatConversation.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useEffect } from 'react'
+
+import { ChatMessages } from '~/components/aiAgent/ChatMessages'
+import { Message } from '~/components/aiAgent/llmOutputs'
+import { OnConversationSubscriptionHookResult } from '~/generated/graphql'
+import { ChatRole } from '~/hooks/aiAgent/aiAgentReducer'
+import { useAiAgent } from '~/hooks/aiAgent/useAiAgent'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+
+interface ChatConversationProps {
+  subscription: OnConversationSubscriptionHookResult
+}
+
+export const ChatConversation: FC<ChatConversationProps> = ({ subscription }) => {
+  const { lastAssistantMessage, state, setChatDone, streamChunk } = useAiAgent()
+  const { translate } = useInternationalization()
+
+  useEffect(() => {
+    if (subscription.data?.aiConversationStreamed.chunk) {
+      if (lastAssistantMessage) {
+        streamChunk({
+          messageId: lastAssistantMessage.id,
+          chunk: subscription.data.aiConversationStreamed.chunk,
+        })
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [subscription.data?.aiConversationStreamed.chunk])
+
+  useEffect(() => {
+    if (lastAssistantMessage && subscription.data?.aiConversationStreamed.done) {
+      setChatDone(lastAssistantMessage.id)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Should we consider removing this rule? I feel like no-one list the full exhaustive list